lib/api/compiler/class-beans-compiler.php

```diff
@@ -732,8 +732,7 @@ private function get_new_hash( $hash, array $fragments_filemtime ) {
 	 * @return void
 	 */
 	private function remove_modified_files( $hash, $filemtime_hash ) {
-		$items = @scandir( $this->dir );  // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged -- Valid use case.
-		unset( $items[0], $items[1] );
+		$items = beans_scandir( $this->dir );
```

Comment:
You used `beans_scandir` instead of `beans_scan_dir` because of the PHP `scandir` function?

Reply:
Yes.  Keeping it similar and familiar.

Reply:
Makes sense 😄 👍